### DISCUSSION TITLE
Upload my student sertifica

### DISCUSSION THREAD

**Discussion by fatihcandvc** [community/community#136569] [GitHub Education]
There is a different way to upload my sertifica in GitHub bucause I don't upload my sertifica There is a rule about taking photos but photos cannot be taken from phones

**Comment by ferid333** [comment]
It seems like you’re asking about a situation where you need to upload a certificate to GitHub, but there are some constraints or rules involved, particularly around taking photos of the certificate with a phone.

Here’s how you could address this situation:

"If you're facing restrictions on uploading your certificate to GitHub or if there are rules against taking photos with a phone, you might consider alternative methods. Instead of uploading a photo, you can:

1. **Scan the Certificate**: Use a scanner or a scanning app on your computer to create a digital copy of the certificate. Scanned images are generally more acceptable and meet higher quality standards compared to phone photos.

2. **Convert to PDF**: Once you have a scanned copy, save it as a PDF. PDFs are commonly accepted on GitHub and are easy to view and share.

3. **Redact Sensitive Information**: If there’s any sensitive information on the certificate, make sure to redact it before uploading. You can use tools like Adobe Acrobat or online PDF editors to do this.

4. **Use GitHub Gists or Repositories**: Depending on how you want to present the certificate, you can either upload it directly to a repository or use GitHub Gists for a more discrete or private upload.

Remember, following the rules and guidelines for uploading documents is important, and these steps can help you comply while still sharing your certificate on GitHub."

This approach addresses the constraints and provides a clear alternative to taking photos with a phone.
